pkg/packer/manifests/gcp/packer.pkr.hcl

```diff
@@ -0,0 +1,363 @@
+packer {
+  required_plugins {
+    googlecompute = {
+      version = ">= 1.0.11"
```

Comment:
should we keep plugin version constraints in one place in `Dockerfile.devkit` that way we wont mismatch version requirements and make sure that required plugins versions are available for air-gapped installations.

Reply:
sync the versions of devkit and hcl...

However when writing HCL this specifies a minimum version which is in this case the one this was developed against. 

Furthermore adding the devkit downloads is only be done because of goss... ideally we will use `packer init` in the future which makes specifying it in the container useless

Reply:
makes sense. thank you.